tests/test_lab.py

```diff
@@ -166,3 +171,32 @@ def test_external_custom_loss_functions(mc):
 
     mc.remove_external_custom_loss(removed_name)
     assert mc.get_variable("NumCustomLossesExternal_Lab") == no_external_losses + 1
+
+
+def test_lab_model_export():
+    mc = MotorCAD(open_new_instance=False)
+    mc.set_variable("MessageDisplayState", 2)
+    mc.load_template("e8")
+    file_path = get_dir_path() + r"\test_files\temp_files\lab_model_export.lab"
+
+    if path.exists(file_path):
+        remove(file_path)
+
+    assert path.exists(file_path) is False
```

Comment:
Let's also check some failure cases in this function. e.g. lab model file not built

We can also try and force one of these other cases. e.g. change lab model build speed:
![image](https://github.com/user-attachments/assets/0f1a4220-2973-48f7-8c2e-c137e3da145f)

and check the error message contains some expected keywords

![image](https://github.com/user-attachments/assets/d3593ce5-9ab2-47b4-8062-713f7175d627)